src/agni/client/receiver/MessageParser.java

```diff
@@ -1,5 +1,24 @@
 package agni.client.receiver;
 
+import agni.client.view.*;
+
 public abstract class MessageParser {
+    private ReceiverListener[] listeners;
+
+    public MessageParser() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
```

Comment:
The `super()` doesn't do anything since it only inherits `Object`. Since this is an abstract class, we should just remove the constructor altogether since this class will never be "constructed".